gemstash.gemspec

```diff
@@ -28,7 +28,12 @@ you push your own private gems as well."
   spec.executables   = spec.files.grep(%r{^exe/}) {|f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "activesupport", "~> 4.2"
+  if RUBY_VERSION > "2.3"
```

Comment:
How about if we make the dependency:
```
spec.add_runtime_dependency "activesupport", ">= 4.2"
```
That way it should work on both older and newer versions of Ruby?